modules/@apostrophecms/user/index.js

```diff
@@ -168,6 +168,11 @@ module.exports = {
   handlers(self) {
     return {
       beforeInsert: {
+        async ensurePassword(req, doc, options) {
```

Comment:
This only needs to happen on insert, right? There is no way to update a user and clear out the password?